### PR TITLE
Documentation

### LINKED ISSUE #4: Contributing document
For maintaining we need to have a rough idea of how people can contribute to the styleguide.

### PR DESCRIPTION
**Pull request by johnwheal**
Fixed #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,52 @@
...
+
+This repository contains all the sass files and associated documentation.
+
+## Found a Bug?
+
+Please raise a GitHub issue for this repository.
+
+## Documentation
+
+The sass files are documented with KSS and are viewable [here](http://port.whealmedia.com). The documentation is automatically generated from the master branch by [Travis](http://tavisci.org).
```
Comment (zoetrope69) [inline README.md]: I'd move the live link to the very top

File: README.md [hunk 2]
```diff
@@ -1,6 +1,52 @@
 # University of Portsmouth Styleguide
 
-### Install Instruction
+This is the University of Portsmouth's living styleguide to be used for internal University projects. It contains CSS and sample HTML and will be used as the basis for future projects.
+
+## Table of contents
```
Comment (zoetrope69) [inline README.md]: Not sure if neccessary

File: sass/README.md [hunk 3]
```diff
@@ -1,3 +1,64 @@
...
+
+Many services within the University are currently using [Bootstrap](https://github.com/twbs/bootstrap) and therefore this styleguide is built on top of it.
+
+This styleguide has been generated with [KSS](https://github.com/kss-node/kss-node) and is automatically built using [Travis](http://travisci.org) on commit to the master branch. You can view its associated GitHub repository [here](https://github.com/University-of-Portsmouth/styleguide).
+
+## How to use
+
+To get started please select a component from the sidebar.
+
+*TBC - Bower?*
```
Comment (zoetrope69) [inline sass/README.md]: Yeah need to work out how it'll be consumable. Would prefer extending the SASS files, but realistically some people may need just a complete file.

I'd avoid Bower, npm is more suitable.

File: sass/README.md [hunk 4]
```diff
@@ -1,3 +1,64 @@
...
+
+## How to contribute
+
+The sass folder contains all the elements of the styleguide. Do not edit the other files or folders - these are used to generate this styleguide.
+
+The sass has the following structure:
```
Comment (zoetrope69) [inline sass/README.md:19]: This will change, but good for now!

File: sass/README.md [hunk 5]
```diff
@@ -1,3 +1,64 @@
-# Information
+# Overview
 
-As many of the services in the University are using Bootstrap, this styleguide is built ontop of Bootstrap's API also.
+Welcome to the University of Portsmouth living styleguide.
```
Comment (zoetrope69) [inline sass/README.md]: Maybe a prompt to read more about living styleguides, pattern libraries etc.

**Comment by zoetrope69** [comment]
I think a CONTRIBUTING.md document would be a good idea too

**Comment by zoetrope69** [comment]
If this repo README.md is for instructions on the styleguide itself, then we can direct people to the usage instructions at the live site/README.md in the sass folder?

File: README.md [hunk 6]
```diff
@@ -1,38 +1,31 @@
...
+### Installation
 
-## Found a Bug?
+1. Install Node >=5.0.0 and npm
+2. Install dependencies: `npm install`
+3. Build sass and styleguide: `npm start`
 
-Please raise a GitHub issue for this repository.
- 
 ## Copyright and license
```
Comment (zoetrope69) [inline README.md:29]: Do we need this if there is a LICENSE file?

Comment (johnwheal) [inline README.md:29]: I think so to make it clear that it's not open source

**Pull request merged**